core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -110,22 +110,22 @@
  */
 
 #define DRAM0_BASE		0x80000000
-#define DRAM0_SIZE		0x80000000
+#define DRAM0_SIZE		0x7F000000
 
 #ifdef CFG_WITH_PAGER
 
 /* Emulated SRAM */
-#define TZSRAM_BASE		(0x06000000)
+#define TZSRAM_BASE		0xFF000000
 #define TZSRAM_SIZE		CFG_CORE_TZSRAM_EMUL_SIZE
 
 #define TZDRAM_BASE		(TZSRAM_BASE + CFG_TEE_RAM_VA_SIZE)
-#define TZDRAM_SIZE		(0x02000000 - CFG_TEE_RAM_VA_SIZE)
+#define TZDRAM_SIZE		(0x01000000 - CFG_TEE_RAM_VA_SIZE)
```

Comment:
I was first thinking about asking for a change of the variable name from `TZDRAM_...` to `TZCDRAM_...`? But on the other hand, it's still DRAM and this change would probably not make things much clearer anyway.